src/lib/components/Elapsed.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+    import * as Tooltip from '$lib/shadcn/components/ui/tooltip';
+    import { elapsed_time } from '$lib/util';
+
+    export let date: Date | number;
+    export let tooltipSide: 'top' | 'left' | 'bottom' | 'right' = 'top';
+
+    const trueDate = new Date(date);
+
+    const time = elapsed_time(trueDate);
+    const dtf = Intl.DateTimeFormat(undefined, {
+        dateStyle: 'long',
+    });
+</script>
+
+<Tooltip.Root openDelay={5}>
+    <Tooltip.Trigger>
+        <div class="text-sm text-slate-500">
+            {time}
+        </div>
+    </Tooltip.Trigger>
+    <Tooltip.Content side={tooltipSide}>
+        <time datetime={trueDate.toISOString()}>
+            {dtf.format(trueDate)}
+        </time>
+    </Tooltip.Content>
+</Tooltip.Root>
```

Comment:
In the UI, the pointer cursor appears on hover. This usually indicates to a user that the element does something when clicked. I'm not sure that's super meaningful here.

The delay also doesn't seem to work. It opens instantly for me.

Reply:
The delay is in milliseconds.  The tooltip component adds the cursor, but I'll see if it can be changed.

Reply:
I had a feeling it might be in MS. It's just quite fast. The cursor definitely bothers me more than the delay